src/common/bad_blocks.c

```diff
@@ -114,8 +114,11 @@ badblocks_get(const char *file, struct badblocks *bbs)
 	if (fd != -1)
 		os_close(fd);
 
-	if (ret && bb_found == -1)
+	if (ret && bb_found == -1) {
 		errno = pmem2_err_to_errno(ret);
+		if (errno == PMEM2_E_NOSUPP)
```

Comment:
huh, is this really possbile? if it is it looks like a bug in pmem2_err_to_errno

Reply:
I thought so, but I am not sure now, because one build still fails, I will verify it ...